x/hold/spec/02_state.md

```diff
@@ -0,0 +1,22 @@
+# State
+
+The `x/hold` module uses key/value pairs to store hold-related data in state.
+
+## Holds
+
+Holds on funds are recorded by address and denom using the following record format:
+
+```
+0x00 | len(<address>) | <address> | <denom> -> <amount>
+```
+
+Where:
+
+* `0x00` is the type byte, and has a value of `0` for these records.
+* `len(<address>)` is a single byte containing the length of the `<address>` as an 8-bit byte in big-endian order.
+* `<address>` is the raw bytes of the address of the account that the funds are in.
```

Comment:
It would be helpful to know how many bytes that each section of the key would need.

Reply:
There's no set length for the address. That's why there's a length byte right before it.